.eslintrc

```diff
@@ -20,6 +22,7 @@
     "react"
   ],
   "rules": {
+    "import/no-duplicates": "off",
```

Comment:
Like the others below, we probably want to enable this at some point - just disabling to avoid changing a zillion files right now.

Reply:
Noted in #607.